app/templates/components/component-tree-item.hbs

```diff
@@ -51,4 +82,4 @@
       {{svg-jar "code-line" width="20px" height="20px"}}
     </button>
   {{/if}}
-</div>
+</div>
```

Comment:
newline 😅

Reply:
Even in hbs? I don't fully understand why we need this.

Reply:
You are right, hbs is excluded from the stock editorconfig so it's fine one way or the other, although it probably doesn't hurt (the reason it's excluded is sometimes you really care that it doesn't have extra whitespace in HTML, but most of the time it doesn't matter). And in this case, you (your editor, presumably?) removed the newline even though you are not editing that and it doesn't seem intentional. This [blog post](https://thoughtbot.com/blog/no-newline-at-end-of-file) may help? It's mostly a convention thing that where a lot of tooling (git, GitHub, unix tools etc) have built around, and comes with some minor benefits also.